packages/wonder-blocks-button/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -58,7 +58,6 @@ exports[`wonder-blocks-button example 1 1`] = `
         "textDecoration": "none",
       }
     }
-    tabIndex={0}
```

Comment:
`tabIndex={0}` is the default order so we don't need to add it.

Reply:
I think we need this. Adding `tabindex=0` to a previously non-focusable element adds that element to the tab navigation order. Removing this caused some elements that used ClickableBehavior, such as `div` and `li`, to no longer be tabbable.

Reply:
I think we have two options. We can leave it out here and tell users of ClickableBehavior that if the component that receives the handlers isn't naturally tabbable, they have to set a `tabIndex`. Else we can include it here and warn users that if they're wrapping an `input`/`a`/`button` type element, they need to make sure that the handlers go onto that element (otherwise they may end up with two tab stops).

Reply:
Good catch!.  This raises an interesting question though about kinds of things we want to add `ClickableBehavior` to.  I'll add it back in for now.

Reply:
See https://github.com/Khan/wonder-blocks/pull/251.